arclightning_backend/src/splash.rs

```diff
@@ -0,0 +1,16 @@
+extern crate rand;
+use rand::Rng;
+use std::fs;
+use std::io;
+
+fn select_random_image(image_dir: PathBuf) -> io::Result<PathBuf>{
+
+    
+    let mut paths: Vec<PathBuf> = fs::read_dir("image_dir").?;
```

Comment:
`.?` is invalid syntax. This should be `fs::read_dir("image_dir")?.flatten().map(|entry| entry.path()).collect()`

Reply:
Since this is only read from, `mut` is unneccessary